bot/exts/fun/hangman.py

```diff
@@ -0,0 +1,121 @@
+from asyncio import TimeoutError
+from pathlib import Path
+from random import choice
+
+from discord import Color, Embed
+from discord.ext import commands
+
+from bot.bot import Bot
+
+TOP_WORDS_FILE_PATH = Path("bot/resources/fun/top_1000_used_words.txt").resolve()
+# defining a list of images that will be used for the game to represent the hangman person
+IMAGES = [
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472355056295946/hangman0.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472756744814613/hangman1.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472808699629578/hangman2.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472862441267230/hangman3.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472950991396864/hangman4.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883472999431430204/hangman5.png",
+    "https://cdn.discordapp.com/attachments/859123972884922418/883473051277226015/hangman6.png"
```

Comment:
PEP 8 suggests to use the trailing comma (for a bunch of reasons, such as minimizing diffs, easy maneuvering, and symmetry), so a comma is missing here after the string.

Reply:
We don't follow that style, I think it is fine with or without it.

Reply:
Really? I believe Kat had a discussion with us about that.

Reply:
[From Kat:](https://github.com/python-discord/sir-lancebot/pull/760#issuecomment-853872766)
> Please see what PEP8 has to say about trailing commas: https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas. Our style guidelines falls back to PEP8 and regardless trailing commas are good since we use version control and they ensure the git blame is accurate.